src/users/users.service.ts

```diff
@@ -58,15 +59,51 @@ export class UsersService {
         await this.loginRepo.delete({employee: user});
         return await this.usersRepository.delete(user);
     }
-    
+    // TEMPORARY ONLY
+    // Create Dummy if Database is empty.
+    // This endpoint will add admin Dummy
+    async createDummy(): Promise<Users> {
+        const user = new Users();
+        user.role = Role.Admin;
+        user.employeeId = 0;
+        user.firstName = 'dummy';
+        user.lastName = 'dummy';
+        user.isManager = true;
+        user.positionTitle = 'dummy';
+        user.startDate = new Date("2014-12-18");
+
+        let company = await this.companyservice.createCompany({
+            companyId: 1, 
+            name: 'dummy', 
+            tags: undefined, recognitions: undefined,
+            users: undefined
+        });
+        user.company = company
+
+        const login = new Login();
+        login.email = 'dummy';
+        login.password = 'dummy';
+        login.employee = await this.usersRepository.save(user);
+        await this.loginRepo.save(login);
+        return user
+    }
+
     async createUser(createuserDto: Users & Login & {managerId: number} & {companyName: string}): Promise<Users> {    
         const user = new Users();
         if (createuserDto.company != undefined) {
             user.company = createuserDto.company;
+
         }
         else{
             if (createuserDto.companyId != undefined) {
                 let company = await this.companyRepository.findOne({where:{companyId: createuserDto.companyId}})
+                // If company.name to companyName if they are not the same
+                // Don't 
+                if (company.name != createuserDto.companyName){
+                    company.name = createuserDto.companyName; 
+                    await this.companyRepository.save(company)
+                }
+
                 if (!company ) {
```

Comment:
Maybe if there is no company we should return an error rather than create a company? This might be a design decision we should discuss with others though. @smalhotra3599 @jeremydrn @jarodroto @matt-weiner @cubscout61 What do you guys think?

Reply:
Since creating a user might be a regular admin role, while creating a company will be a dev/server admin role, we might wan't to just return an error if a company doesn't exist.